scripts/west-commands.yml

```diff
@@ -34,3 +34,8 @@ west-commands:
       - name: ncs-genboard
         class: NcsGenboard
         help: generate board skeleton files for any Nordic SoC
+  - file: scripts/west_commands/provision-54l.py
+    commands:
+      - name: provision54l
```

Comment:
Can we please rename this to `ncs-provision` and then pass a parameter with the SoC family, so we are prepared for future ICs?